book/src/concepts/entity_and_component.md

```diff
@@ -0,0 +1,105 @@
+# Entity and Component
+
+## What are `Entity` and `Component` ?
+
+An `Entity` conceptually represents a single object in your world. `Component` represents one aspect of an object. For example, a bottle of water has a shape, a volume, a color and is made of a material (usually plastic). In this example, the bottle is the entity, and the properties are components.
```

Comment:
"conceptually" is superfluous in this sentence. We can also use "property" instead of "aspect" earlier as well.

Reply:
Agree with "conceptually". I deleted it :)
I don't like the word "property" here because its meaning gets too overloaded. Aspect feels like a good choice because it has a broader scope and might encourage people to group related information into a single component. Property might easily be misread as encouraging the opposite.